openpype/hosts/max/plugins/publish/extract_thumbnail.py

```diff
@@ -0,0 +1,122 @@
+import os
+import tempfile
+import pyblish.api
+from openpype.pipeline import publish
+from pymxs import runtime as rt
+
+
+class ExtractThumbnail(publish.Extractor):
+    """
+    Extract Thumbnail for Review
+    """
+
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Thumbnail"
+    hosts = ["max"]
+    families = ["review"]
+
+    def process(self, instance):
+        self.log.info("Extracting Thumbnail ...")
+
+        # TODO: Create temp directory for thumbnail
+        # - this is to avoid "override" of source file
+        tmp_staging = tempfile.mkdtemp(prefix="pyblish_tmp_")
+        self.log.debug(
+            f"Create temp directory {tmp_staging} for thumbnail"
+        )
```

Comment:
Doesn't that just mean we'd need to change the name of the `filename` below? e.g. `filename = f"{name}_thumbnail.jpg"`?